### PR TITLE
Updating docs for Rails 5.0

### PR DESCRIPTION
**Pull request by lucascaton**

### CODE REVIEW

File: guides/source/command_line.md [hunk 1]
```diff
@@ -395,10 +395,10 @@ INFO: You can also use `rake -T`  to get the list of tasks.
...
 $ bin/rake about
 About your application's environment
-Rails version             4.2.0
-Ruby version              1.9.3 (x86_64-linux)
-RubyGems version          1.3.6
-Rack version              1.3
+Rails version             5.0.0
+Ruby version              2.2.0 (x86_64-linux)
+RubyGems version          2.4.5
+Rack version              1.5
```
Comment (carlosantoniodasilva) [inline guides/source/command_line.md]: Rails is using [rack 1.6](https://github.com/rails/rails/blob/master/actionpack/actionpack.gemspec#L24), this should probably reflect that.

Comment (lucascaton) [inline guides/source/command_line.md]: Thanks @carlosantoniodasilva, I'll update this.

**Comment by seuros** [comment]
There is a lot of cosmetic changes. Could you remove them and squash your commits ?

**Comment by lucascaton** [comment]
@seuros like removing trailing whitespace?

**Comment by seuros** [comment]
Yes and the extra spaces in helpers.rb
We could probably make an exception.
cc @rafaelfranca  wdyt ?

**Comment by rafaelfranca** [comment]
Yeah, helpers change is not necessary. The rakefile changes are fine.

About the patch I'd not include the release notes right now. We decided to
just write it before the release because the live document approach we took
at 4.2 didn't worked. So it is better to no create the file.

**Comment by lucascaton** [comment]
@seuros @rafaelfranca Sure, no problem. I've created another PR: https://github.com/rails/rails/pull/18290.

**Pull request closed without merging**